README.md

```diff
@@ -26,98 +26,47 @@ and render the view.
-1. name: dish name.
-2. description: a description of the dish.
-3. dish_type: entree, dessert, etc. We really didn't have to get nit picky with this.
-### 2. Ingredients ###
-Has the ingredients. Each Ingredient should only appear once in here. Has a many to many relationship with dishes through recipes.
-1. name: ingredient name.
-2. everything else: we kept it from the last group's work. We don't know how they are useful, although it may become relevant once you see the recipes for the dishes used in MayFlower You can delete them if you see fit.
-### 3. Menus ###
-Is the main thing show to end users. The Menus is on a 49 day cycle (7 days, 7 weeks). Each day has breakfast, dinner (lunch as the younger kids -- <50 --  call it), and supper (dinner as the younger kids call it). Has a many to many relationship with dishes through dishes_menus.
-1. day: A number between 1-49. Essentially the day in the cycle.
-2. type_of_meal: Breakfast, dinner, or supper.
-### 4. Recipes ###
-Connects dishes to ingredients. Also holds some extra information about their connection. Keep in mind that a single recipe entry is a step in real world recipes. Hence, a dish will have multiple recipes, each of which will use a different ingredient.
-1. comment: it's the instruction of this particular step of the recipe.
-2. step: it's an integer specifying the order at which the recipe entries should be done.
-3. portion_size: how much of an ingredient should be used.
-4. ingredient_id: the id of the ingredient used in this step.
-5. dish_id: the id of the dish this recipe entry belongs to.
-### 5. Dishes_Menus ###
-Connects dishes to menus and contains some information about their relationship.
-1. dish_id: the dish that should be connected to a particular menu.
-2. menu_id: the menu the dish should be connected to.
-3. temp: a boolean that specifies that this entry is temporary. The default is false. This column allows temporary additions to a menu for this cycle. In the next cycle, this connection should be deleted.
-4. show: a boolean that specifies that this entry should be shown in the final menu. The default is true. This column allows temporary omissions from a menu. In the next cycle, this connection should still be there, but it should be set to true if it was false before.
-
-Finally, the scheduled command that resets all temp/show changes to in the dishes_menus is defined as the rake task here: lib/tasks/scheduled_db_commands.rake, and you should use heroku scheduler to run it in production. It required extra steps from the owner of the repo (professor), so we didn't add that. See: https://devcenter.heroku.com/articles/scheduler for how to set it up.
+Has the dishes. Each dish should only appear once in here. Has a one to many relationship with entires in dishes_menus.
+1. id: dish id. Auto-incremented.
+2. name: dish name.
+3. star: 0-3. 0 for no star. 1 for \*. 2 for \*\*. 3 for \*/\*\*.
```

Comment:
We NEED to explain what stars mean, as it is not self evident. Add something like
"The number of stars represents which, if any, special menus (for people who can't choose their own meals, or otherwise) a dish is on. 0 stars indicate it is on no special menus, 1 star for the dinner (lunch) special menu, 2 stars for the supper (dinner) special menu and 3 stars for both dinner and supper special menus"
Writing this makes me wonder; should star be an attribute of DishMenu? Because what if Applesauce is on a special menu one day, but not another? Just a thought.

Reply:
I've also wondered about that. I remember after asking Scott we arrived at the conclusion that it should belong to dishes.